fargate/log-puller.yml

```diff
@@ -0,0 +1,173 @@
+# adapted from https://github.com/nathanpeck/aws-cloudformation-fargate/
+
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Deploy log-puller on AWS Fargate, hosted in a public subnet, and accessible via a public load balancer.
+Parameters:
+  StackName:
+    Type: String
+    Default: fargate-public-network
+    Description: The name of the parent Fargate networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+  ServiceName:
+    Type: String
+    Default: log-puller
+    Description: A name for the service
+  ImageUrl:
+    Type: String
+    Default: concordconsortium/log-puller
+    Description: The url of a docker image that contains the application process that
+                 will handle the traffic for this service
+  ContainerPort:
+    Type: Number
+    Default: 5000
+    Description: What port number the application inside the docker container is binding to
+  ContainerCpu:
+    Type: Number
+    Default: 256
+    Description: How much CPU to give the container. 1024 is 1 CPU
+  ContainerMemory:
+    Type: Number
+    Default: 512
+    Description: How much memory in megabytes to give the container
+  Path:
+    Type: String
+    Default: "/log-puller*"
+    Description: A path on the public load balancer that this service
+                 should be connected to. This should NOT be *.
+  Priority:
+    Type: Number
+    Default: 1
+    Description: The priority for the routing rule added to the load balancer.
+                 This only applies if your have multiple services which have been
+                 assigned to different paths on the load balancer.
+  DesiredCount:
+    Type: Number
+    Default: 2
+    Description: How many copies of the service task to run
+  RdsDatabaseUrl:
+    Type: String
+    Description: Url to RDS instance, in format postgres://USERNAME:PASSWORD@HOSTNAME/DATABASE
```

Comment:
Usually there also needs to be a parameter for the RDS Security group, then the template adds an ingress rule so the service can connect to RDS.